cer-graphql/index.js

```diff
@@ -132,7 +132,7 @@ async function createServer(config) {
                 var requestedFields = []; // List of fields requested, populated in the recursive function below
 
                 let recursive_iteration_count = 0; // Tracks the number of recursive iterations
-                const MAX_RECURSIVE_ITERATIONS = 500; // The maximum allowed number of recursive iterations
+                const MAX_RECURSIVE_ITERATIONS = 2000; // The maximum allowed number of recursive iterations
```

Comment:
How dare you update my arbitrary number with another arbitrary number :O

Reply:
its bigger and better